### PR TITLE
Build UI

### PR DESCRIPTION
**Pull request by h0rhay**
Begin the build for the User Interface.

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**S-personal-shopper-landing-page-main**

:tada: Your build was successful! [See the Deploy preview here.](https://spersonalshopperlandingpagemai-buildui.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/49cd8773-d86a-4788-9810-670ddc80a14c/sites/8efc78ca-28d9-43e0-af1e-eea287dcfb65/builds/4f448cdc-c335-4b51-ae76-31506820ba0c/details)

:clock1: Build time: **2m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :green_heart: 100   |
| Accessibility |   :green_heart: 97   |
| Best Practices |   :green_heart: 93   |
| SEO |   :green_heart: 92   |

:link: [View full report](https://build-4f448cdc-c335-4b51-ae76-31506820ba0c.gtsb.io/reports/lighthouse/index.html)

File: src/components/header.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
 import React from 'react';
-import styled from '@emotion/styled';
-import { css } from '@emotion/core';
+import styled from 'styled-components';
 import { Link } from 'gatsby';
 
+const Header = styled.header`
```
Comment (KarthikMuthumari17) [inline src/components/header.js]: 🙃 We can call this as `Wrapper` and `HeaderSection` as `Header`

Comment (h0rhay) [reply src/components/header.js]: I don't really mind, I am a big fan of calling styled component elements the same thing as their html counterpart when using them that way though.
E.g. `Header` = `<header/>`
`Footer` = `<footer/>`

Comment (KarthikMuthumari17) [reply src/components/header.js]: cool 👍

**Review by KarthikMuthumari17 — APPROVED** [review]

**Pull request merged**